src/openrct2/paint/Painter.cpp

```diff
@@ -185,6 +188,17 @@ void Painter::ReleaseSession(PaintSession* session)
     _freePaintSessions.push_back(session);
 }
 
+bool Painter::ShouldShowFPS()
```

Comment:
There is no reason to expose this function in the header. Just make it static and move it above PaintFPS().